src/frontend/apps/bbb/DashboardMeeting/index.tsx

```diff
@@ -0,0 +1,159 @@
+import { Box, Spinner } from 'grommet';
+import React, { lazy, useState, Suspense } from 'react';
+import { toast } from 'react-hot-toast';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+
+import { ErrorMessage } from 'components/ErrorComponents';
+import { getDecodedJwt } from 'data/appData';
+import { Loader } from 'components/Loader';
+
+import { bbbAppData } from 'apps/bbb/data/bbbAppData';
+import { useJoinMeeting, useMeeting } from 'apps/bbb/data/queries';
+const DashboardMeetingStudent = lazy(
+  () => import('apps/bbb/DashboardMeetingStudent'),
+);
+const DashboardMeetingInstructor = lazy(
+  () => import('apps/bbb/DashboardMeetingInstructor'),
+);
+const DashboardMeetingAskUsername = lazy(
+  () => import('apps/bbb/DashboardMeetingAskUsername'),
+);
+const DashboardMeetingJoin = lazy(
+  () => import('apps/bbb/DashboardMeetingJoin'),
+);
+
+const messages = defineMessages({
+  loadingMeeting: {
+    defaultMessage: 'Loading meeting...',
+    description:
+      'Accessible message for the spinner while loading the meeting in dashboard view.',
+    id: 'component.DashboardMeeting.loadingMeeting',
+  },
+  startMeetingSuccess: {
+    defaultMessage: 'Meeting started.',
+    description: 'Message when meeting start is successful.',
+    id: 'component.DashboardMeeting.startMeetingSuccess',
+  },
+  startMeetingFail: {
+    defaultMessage: 'Meeting not started!',
+    description: 'Message when meeting start failed.',
+    id: 'component.DashboardMeeting.startMeetingFail',
+  },
+});
+
+const DashboardMeeting = () => {
+  const intl = useIntl();
+  const canUpdate = getDecodedJwt().permissions.can_update;
+
+  const [meetingUrl, setMeetingUrl] = useState('');
+  const [askUsername, setAskUsername] = useState(false);
+  const [meetingJoined, setMeetingJoined] = useState(false);
+  const [userFullname, setUserFullname] = useState(
+    getDecodedJwt().user?.user_fullname || '',
+  );
+
+  const { data: meeting, status: useMeetingStatus } = useMeeting(
+    bbbAppData.meeting!.id,
+    { refetchInterval: 5000 },
```

Comment:
Is it really needed ? If I remember, by default the library will fetch the endpoint every time the focus is taken in the document. If we have thousands of student polling our API every 5 seconds, it can be huge !

Reply:
Yes, focus makes refetch by default.
But I don't think students will try to click on iframes while waiting for the meeting to start.

We can increase poll time, anyway.

Reply:
Yes, I stopped it for students only, as instructor can see some stats about the meeting in his dashboard.